modules/ossm-sidecar-validate-kiali.adoc

```diff
@@ -0,0 +1,23 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc
+////
+:_content-type: CONCEPT
+[id="ossm-validating-sidecar_{context}"]
+= Validating sidecar injection
+
+The Kiali console offers several ways to validate whether or not your applications, services, and workloads have a sidecar proxy.
+
+The *Graph* page displays a node badge indicating a *Missing Sidecar* on the following graphs:
+
+* App graph
+* Versioned app graph
+* Workload graph
+
+The *Applications* page displays a *Missing Sidecar* icon in the *Details* column for any applications in a namespace that do not have a sidecar.
```

Comment:
I miss screenshots. It's hard to follow these without a picture.

The Kiali project is a dashboard / User Interface and not having any pictures is something odd to me.

Reply:
Yeah, I think the docs team needs to have a discussion around how to add at least a few screenshots.  The problem is, they have to be localized, so they need to be reproducable for the translation team.

Reply:
I can attach other examples from Openshift documentation where there are screenshots with draconian rules